src/components/AgmExtension/AgmExtensionEvaluation.vue

```diff
@@ -84,21 +83,7 @@
           cols="12"
           sm="9"
         >
-          <template v-if="!data.isEligible">
-            <template v-if="data.alreadyExtended && !data.requestExpired">
-              The due date for this AGM cannot be set since extension has already been requested.
-            </template>
-            <template v-else-if="!data.alreadyExtended && data.requestExpired">
-              The due date for this AGM cannot be set since the request for extension has expired.
-            </template>
-            <template v-else-if="data.alreadyExtended && data.requestExpired">
-              The due date for this AGM cannot be set since extension has already been requested and the request
-              for extension has expired.
-            </template>
-          </template>
-          <template v-else>
-            {{ dueDateString }}
-          </template>
+          {{ dueDateString }}
```

Comment:
I just noticed that `dueDateString` was created using DateUtilities.formatYyyyMmDd(). However, this method creates a date with an abbreviated month, while the uxpin asks for a long month.

I think you'll have to do the same thing I do here (or create a new utility method and use it in both your and my code):
https://github.com/bcgov/business-filings-ui/blob/78c7ad64e868a4d9682bd6b9cbad004e0fc3843c/src/components/Dashboard/FilingHistoryList/filings/AgmExtension.vue#L39